### DISCUSSION TITLE
Moxy Integration

### DISCUSSION THREAD

**Discussion by Hockje** [dvmarinoff/Auuki#131] [Ideas]
Would it be possible to pull the bluetooth signal from a moxy monitor and have it display on the dashboard? 

Does this data pull into the fit file?

**Comment by dvmarinoff** [comment]
I am not really familiar with this class of devices.
My quick look up shows that the Moxy sensors are transmitting over ANT+ and Bluetooth.

### ANT+
it uses a standard and documented protocol. But Flux does not yet have a stable support for ANT+, so this is a no go at the moment.

### Bluetooth
I can see that the official Moxy settings app is using web-bluetooth (the same as Flux) which means that support is technically possible, but I can’t find a specification document about the bluetooth service the Moxy is using. I’ll have to contact them to see if they are willing to share the service specification.

### .FIT
.FIT is supported so no issues here, I just need to add the corresponding fields to my implementation

**Comment by dvmarinoff** [comment]
@Hockje 

Just letting you know, I successfully entered into communication with the people at Moxy.
They are very nice and expressed full support. Development is in progress. In a few weeks I think I'll be able to ship support for a minimal set of features. I will keep posting updates about the progress into the dedicated issue here: https://github.com/dvmarinoff/Flux/issues/132

Meanwhile I need to figure out what user interface will work best and I will need to better educate myself on how the device is being used by people in practice. It will be very helpful if you can describe how are you using the Moxy and which data points are most important to you?

**Comment by Hockje** [comment]
Hello, 

I think its best to have the ThB and the %Smo2 values. I know there is some garmin data field that use the average of the lap. MIght be a cool feature. I like that your software is simple and easy to work. I use to use trainer road and PerfPro and this is a simlar vibe. I like it alot. 

Thank you!
